README.md

```diff
@@ -12,31 +14,27 @@ A plugin style architecture is used which allows developers to leverage their ow
 This pattern aims to promote flexibility, reusability and extensibility of different Adaptive Card templates and plugins across development teams. The [Design Principles] section goes into more details.
 
 [adaptive card]: https://docs.microsoft.com/en-us/adaptive-cards/
```

Comment:
NIT: Is there a reason why these sections (eg: linking to adaptive card) are never capitalized even if in the text we use them in a sentence as capitalized? Would it look tidier if we were consistent?

Reply:
There's not really a strict reason for this, the links work either way and they're only displayed how they appear in the text they're referenced in. Still, I agree it would be cleaner to have them be more consistent -- even in the sections 
defining the links -- so I'll make sure they're consistent

Reply:
I didn't realize this before, but this actually seems to be something that our linters are enforcing. All of these linking sections are set to be all lowercase when we save changes